db/models/orders.js

```diff
@@ -1,14 +1,41 @@
 const client = require("../client.js");
 
+// helper function
+const addProductsToOrders = async (orders) => {
+  try {
+    const orderIdArray = orders.map((order) => {
+      return order.id;
+    });
+    // orderProductId created for the frontend
+    const { rows: products } = await client.query(`
+                  SELECT products.*, order_products.quantity, order_products.price, order_products."orderId", order_products.id AS "orderProductId"
+                  FROM products
+                  JOIN order_products
+                  ON products.id = order_products."orderId"
```

Comment:
our JOIN should be ON products.id = order_products."productId"

Reply:
Done.